specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/networkManagerGroup.json

```diff
@@ -0,0 +1,616 @@
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/networkGroups/{networkGroupName}/staticMembers/{staticMemberName}": {
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/StaticMemberNameParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "StaticMembers"
+        ],
+        "operationId": "StaticMembers_Get",
+        "description": "Gets the specified static member.",
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the static member.",
+            "schema": {
+              "$ref": "#/definitions/StaticMember"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "StaticMembersGet": {
+            "$ref": "./examples/NetworkManagerStaticMemberGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "StaticMembers"
+        ],
+        "operationId": "StaticMembers_CreateOrUpdate",
+        "description": "Creates or updates a static member.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StaticMember"
+            },
+            "description": "Parameters supplied to the specify the static member to create"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated - Returns information about the static member.",
+            "schema": {
+              "$ref": "#/definitions/StaticMember"
+            }
+          },
+          "201": {
+            "description": "Created - Returns information about the static member.",
+            "schema": {
+              "$ref": "#/definitions/StaticMember"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "StaticMemberPut": {
+            "$ref": "./examples/NetworkManagerStaticMemberPut.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
dont we also need force delete here? static member may be part of an active config

"parameters": [
          {
            "$ref": "#/parameters/ForceDeleteParameter"
          }
        ],

Reply:
I am not familiar with static member apis, can you drive the changes on it?

Reply:
we do not offer force delete here; will leave as-is